Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -1,8 +1,8 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2023 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
-// This file is autogenerated. Please do not modify it.
```

Comment:
This line shouldn't go away. How did you add the new keys, by running the `Localization` scheme?

Reply:
Added it back. Thanks.
And yes, I ran the Localization scheme and included only the changes related to this feature.